testing/visual-regression/backstop-common.js

```diff
@@ -116,6 +116,12 @@ module.exports = function backstopCommon(baseUrl) {
         selectors: ['.cads-breadcrumbs'],
       },
 
+      // Forms
+      {
+        label: 'Forms/Error summary',
+        url: storyUrlFor('forms-error-summary--example'),
```

Comment:
focus state for summary div and links?

Reply:
Good shout, will combine that with focus state tests for the other form elements as a follow on PR